### ISSUE TITLE
RNN

### ISSUE THREAD

**Issue by ronitkathuria15** [ronitkathuria15/Handrawn-image-recognition#2]
A recognizer capable of building a larger dataset consisting of more categories can be developed through the use of a Recurrent Neural Network (RNN). This includes the use of a combination of convolutional layers, LSTM layers, and a softmax output layer to classify the drawings.
